odo/tests/test_append.py

```diff
@@ -12,3 +12,16 @@ def test_append_list_to_set():
     s = set([1, 2, 3])
     append(s, [4, 5, 6])
     assert s == set([1, 2, 3, 4, 5, 6])
+
+
+def test_append_set_to_list():
+    s = set([3, 4, 5])
+    l = [1, 2, 3]
```

Comment:
minor bikeshedding comment, on some typefaces `1` and `l` are the same glyph, could we use names like `set_` and `list_` here? Note the suffixing underscore to not shadow the built in names.